recipes/nbcelltests/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set name = "nbcelltests" %}
+{% set version = "0.1.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 904656317adb86faba5cb5640195e79ea5022eab2ab2852a53975b5346f3659d
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python >=3.7
```

Comment:
The wheel on pypi also work with py36. Was the minimum constraint on 3.7 intentional here (and below)?